### PR TITLE
Add styling to the time selection UI and add preview gif to the readme

### LINKED ISSUE #13: Enhance the UI for selecting the time

### PR DESCRIPTION
**Pull request by R-Jin**
Here is a preview of the styling (This is also the gif that i put in the readme):
extension

### CODE REVIEW

**Review by drgn-hart — COMMENTED** [review]
Thanks for working on this issue!! It's really appreciated.

Can you please do the below changes:
1. Add a new directory called **assets** and move the extension.gif into that directory.
2. Update README.md appropriately.
3. Can we add hh:mm text in the clock so that user is clear that it is in that format. (**Note:** Not an html expert, so please check and let me know if this is not feasible.)

Let me know if you have any questions.

**Comment by R-Jin** [comment]
Will do

**Comment by R-Jin** [comment]
@drgn-hart 
I am not sure how to make a placeholder inside the input with the type time and when reading about it online it seems like it is not possible due to the input beeing browser dependent.

With all of this said an alternative is to create 2 dropdown inputs and get the data from them. With these I could put in a placeholder text I think.

**Comment by R-Jin** [comment]
After some further digging I found out that it is indeed possible to do it.

File: src/core/timer.css [hunk 1]
```diff
@@ -7,4 +7,6 @@
     height: 40px;
     text-align: center;
     font-size: x-large;
+    padding-top: 10px;
```
Comment (drgn-hart) [inline src/core/timer.css]: I was testing these changes locally and this padding seems to have added bit weird placing of the timer on few pages. Please find the screenshot below:
image
I tried it on github.com and leetcode.com and this came out this way. However, when I tried it on google.com this appeared similar to the gif. Can you check this out?

Comment (R-Jin) [reply src/core/timer.css]: Yes

Comment (R-Jin) [reply src/core/timer.css]: I fixed this issue!

**Review by drgn-hart — APPROVED** [review]

**Comment by drgn-hart** [comment]
@R-Jin  Thanks for your contribution!!

**Pull request merged**